<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterlisImport</class>
 <widget class="QDialog" name="InterlisImport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>840</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Files</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="description">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Please add all your files and repository models you want to use for your project (you can use drag'n'drop).</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="the_frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="input_line_label">
        <property name="text">
         <string>Model or Datafile</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QToolButton" name="add_button">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../../../qgis/QGIS/images/themes/default/symbologyAdd.svg</normaloff>../../../../qgis/QGIS/images/themes/default/symbologyAdd.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="remove_button">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../../../qgis/QGIS/images/themes/default/symbologyRemove.svg</normaloff>../../../../qgis/QGIS/images/themes/default/symbologyRemove.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>674</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="CompletionLineEdit" name="input_line_edit" native="true"/>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QListView" name="source_list_view"/>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="file_browse_button">
        <property name="toolTip">
         <string>Browse Interlis files</string>
        </property>
        <property name="text">
         <string>â€¦</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QTextBrowser" name="txtStdout">
     <property name="toolTip">
      <string/>
     </property>
     <property name="openLinks">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CompletionLineEdit</class>
   <extends></extends>
   <header>QgisModelBaker.gui.options</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>file_browse_button</tabstop>
  <tabstop>add_button</tabstop>
  <tabstop>remove_button</tabstop>
  <tabstop>source_list_view</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
